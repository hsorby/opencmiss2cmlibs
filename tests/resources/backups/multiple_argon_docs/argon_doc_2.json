{
  "OpenCMISS-Argon Version": [
     "0",
     "4",
     "2"
  ]
}