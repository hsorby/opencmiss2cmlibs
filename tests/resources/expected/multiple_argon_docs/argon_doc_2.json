{
  "CMLibs Argon Version": [
    "0",
    "4",
    "2"
  ]
}